uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -11,35 +11,32 @@ define([
   'leave-absences/shared/models/entitlement-model',
   'leave-absences/shared/models/leave-request-model',
   'leave-absences/shared/models/public-holiday-model',
-  'leave-absences/shared/models/instances/leave-request-instance',
-  'leave-absences/shared/models/instances/sickness-leave-request-instance',
-], function (components, _, moment) {
+], function (controllers, _, moment) {
```

Comment:
1. In all other files `bind` has been used so we can keep the consistency here too.
2. If you are really going for saving this into a separate variable, it is better to name 'self', as that is more of an convention.